package.json

```diff
@@ -12,8 +12,8 @@
     "dist": "cross-env NODE_ENV=production webpack --config webpack.config.js",
     "preinstall": "npm i webpack-cli",
     "postinstall": "npm run dist",
-    "test": "mocha test/helpers/browser.js test/unit/**/*.test.js",
```

Comment:
Maybe the glob needs to be in quotes?

Reply:
I can try that.

Reply:
It's funny because it works perfectly on my local, but has a very different outcome in the Travis runner

Reply:
So, I gave this a shot, and it half works in Travis' runner. Unfortunately, now with the updated approach, there is a test for the Feedback Button which fails because it now exceeds the default timeout set by mocha. 

If it is the interest of the team to approach test files in this manner, I propose we ticket the issue for a later date and merge this PR. The refactor of the testing directory structure can be its own task.